recipes/bacnet-stack/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "20200218":
+    sha256: 1fee2f943c2d3e2cbb3a2f9ff3de40b392418c96c751123c054530cc29506b3a
+    url: https://github.com/bacnet-stack/bacnet-stack/archive/677f528aa46bebe5b7a129342b07fc491acd32b1.tar.gz
```

Comment:
```suggestion
    url: https://github.com/bacnet-stack/bacnet-stack/archive/bacnet-stack-0.8.6.tar.gz
```

Reply:
I guess this wont work as this version is 2 years old and has no cmake support

Reply:
Nor class platform support

Reply:
Ah, good point. Then disregard this suggestion!